pulsar-proxy/src/test/java/org/apache/pulsar/proxy/server/ProxyParserTest.java

```diff
@@ -110,7 +117,7 @@ public void testProducer() throws Exception {
         PulsarClient client = PulsarClient.builder().serviceUrl(proxyService.getServiceUrl())
                 .build();
         Producer<byte[]> producer =
-                client.newProducer(Schema.BYTES).topic("persistent://sample/test/local/producer-topic")
+                client.newProducer(Schema.BYTES).topic("persistent://sample/local/producer-topic")
```

Comment:
Why did you modify many namespaces from V1 to V2 in tests? (`sample/test/local` to `sample/local`) It seems irrelevant with the core changes.

Reply:
Yes, I agree.

Reply:
I'll push a force push where these topic name V1 related changes are reverted.